Makefile

```diff
@@ -1,5 +1,9 @@
 # Current Operator version
-VERSION ?= 0.1.3
+# The value here should be equal to the next version:
+# - next minor version on main branch
+# - next patch version on release branches
+VERSION ?= 0.12.0
```

Comment:
I would vote for setting this variable from git tag, but it can be done in next PR.
/lgtm

Reply:
Good idea.